applications/surgical_scene_recon/training/gsplat_train.py

```
@@ -0,0 +1,2208 @@
+            
+            # 1. L1 loss (with masks)
+            # Need to permute for loss functions: [B, H, W, 3] → [B, 3, H, W]
+            image_rendered_p = image_rendered.permute(0, 3, 1, 2)
+            image_gt_p = image_gt.permute(0, 3, 1, 2)
+            mask_p = mask.unsqueeze(1)  # [B, 1, H, W]
+            
+            l1_loss_val = l1_loss(image_rendered_p, image_gt_p, mask_p)
+            
+            # 2. Depth loss (endoscopy-specific)
+            depth_rendered_p = depth_rendered.unsqueeze(1)  # [B, 1, H, W]
+            depth_gt_p = depth_gt.unsqueeze(1)  # [B, 1, H, W]
```

Ruff (0.14.5)

Local variable `depth_gt_p` is assigned to but never used

Remove assignment to unused variable `depth_gt_p`

(F841)